src/app/services/http/forgot-pw-http-service/forgot-pw-http.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from "@angular/core";
+import { Constants } from "@common/constants/constants";
+
+import { BaseHttpService } from "../base-http-service/base-http.service";
+import { AlertTypeEnum } from "@common/enums/alert-type.enum";
+import { PagesUrlsEnum } from "@common/enums";
+
+/**
+ * Service for handling forgot password HTTP requests.
+ * Extends the BaseHttpService to provide functionality for sending password reset emails.
+ *
+ * @author @aBlancoC
+ * @extends BaseHttpService<String>
+ * @@Injectable
+ */
+@Injectable({
+    providedIn: 'root'
+})
+
+export class ForgotPwHttpService extends BaseHttpService<String> {
```

Comment:
aqui igual refactoricelo y pongale

```suggestion
export class ForgotPasswordHttpService extends BaseHttpService<String> {
```